environments/staging/staging-hearings-reform-hmcts-net.yml

```diff
@@ -8,7 +8,6 @@ cname:
     record: sdshmcts-stg.azurefd.net
     ttl: 300
   - name: admin
-    ttl: 300
```

Comment:
is this correct?

Reply:
It was added in two times so removed it as a duplicate - which is now:

cname:
  - name: video
    record: sdshmcts-stg.azurefd.net
    ttl: 300
  - name: admin
    record: sdshmcts-stg.azurefd.net
    ttl: 300
  - name: signalr
    record: vh-infra-core-stg.service.signalr.net
    ttl: 300